provenance/provenance-task.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: distroless-provenance
+  annotations:
+    chains.tekton.dev/transparency-upload: "true"
+    description: |
+      Takes in the distroless repo as an input and builds & pushes distroless images.
+spec:
+  results:
+    - name: IMAGES
+  params:
+    - name: PROJECT_ID
+      type: string
+      default: distroless
+    - name: CHAINS-GIT_COMMIT
+      type: string
+    - name: CHAINS-GIT_URL
+      type: string
+      default: https://github.com/GoogleContainerTools/distroless
+  steps:
+    - name: bazel
+      image: gcr.io/cloud-marketplace-containers/google/bazel:3.4.1
+      env:
+      - name: PROJECT_ID
+        value: $(params.PROJECT_ID)
+      - name: REGISTRY
+        value: insecure-registry.default.svc.cluster.local:5000
+      - name: COMMIT_SHA
+        value: $(params.CHAINS-GIT_COMMIT)
+      script: |
+        #!/bin/sh
+        set -ex
+
+        git clone $(params.CHAINS-GIT_URL)
+        cd distroless
+        git checkout $(params.CHAINS-GIT_COMMIT)
```

Comment:
It might be good to make this another step?

Reply:
agreed! i'm going to convert this to a pipeline in another PR, so i'll add it in there.